crates/ruma-client-api/src/sync/sync_events/v3.rs

```diff
@@ -4,7 +4,7 @@
 
 use std::{collections::BTreeMap, time::Duration};
 
-use super::UnreadNotificationsCount;
+use super::{DeviceLists, UnreadNotificationsCount};
```

Comment:
Can you make `DeviceLists` a `pub use` so it's still available publically under this location, now that we're post the 0.7.0 release and can't make breaking changes to stable API as easily?